tests/torchtune/models/clip/test_clip_tokenizer.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+import pytest
+
+from torchtune.models.clip._model_builders import clip_tokenizer
+
+
+class TestCLIPTokenizer:
+    @pytest.fixture
+    def tokenizer(self):
+        return clip_tokenizer()
```

Comment:
this will download the CLIP vocab (2mb) to `~/.cache/torchtune/clip_vocab.txt.gz` (if it doesn't already exist). is this ok to have in a unit test? should i instead add the vocab to `tests/assets/`?

Reply:
It should be an asset